services/core/shared/core/transactionStatistics.js

```diff
@@ -33,8 +33,8 @@ const tableConfig = {
 	schema: {
 		amount_range: { type: 'string' },
 		count: { type: 'integer' },
-		date: { type: 'integer' },
-		id: { type: 'string' },
+		date: { type: 'integer', isDefaultColumn: true },
+		id: { type: 'string', isDefaultColumn: true },
 		type: { type: 'string' },
 		volume: { type: 'bigInteger' },
 	},
```

Comment:
Let us please move the schema definition to a separate file.
Please consider `services/core/shared/core/schemas/transactionStatistics.js`.